Makefile

```diff
@@ -121,9 +120,11 @@ symlink-core-for-cli:
 
 # Minimal build of the semgrep-core executable. Intended for the docker build.
 # Requires the environment variables set by the included file above.
+# Builds only the two main binaries mainly needed for development. If you
```

Comment:
Can remove one of the main

Reply:
done in 4165f8ea650cbe358af250c44b763c4af45e9086